chennai.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: default
+header: Reaching the Venues
+title: Reaching the Venues
+bloglike: true
+---
+
+This page is to help the attendees reach Chennai and the conference venues.
+
+<br>
+
+### Getting to PyCon India 2019
+
+**By Plane**
+
+Chennai has an international airport that is well connected to Europe, the Middle East, US,
+and South East Asia. It is also well connected by domestic flights.
+
+**Chennai Trade Center(Conference)**
+
+The easiest option is to take a metro from Chennai Airport towards Washermenpet Metro and alight
+at Guindy Metro. From the metro station, you can board one of these buses - 49, 54, 54D and
```

Comment:
Metro towards CMBT should be more appropriate? And the Alandur Metro is the closest to the Chennai Trade Centre.

Reply:
If they get down at guindy metro they have two option, one is walk towards race course bus stop to catch 54, 54D or 49 else they need to walk towards Guindy Railway station bustop which at MKN road. Actually bus comes from race course will come via MKN road

Reply:
@bravegnu, @gnurenga is right. This is why I preferred Guindy metro over Alandur. There is a lot less walking involved. Crossing Kathipara and reaching the right bus stop is confusing too.